### ISSUE TITLE
Sweep early stopping hyperband not work[CLI] 

### ISSUE THREAD

**Issue by hainanxzg** [wandb/client#1975]
wandb --version && python --version && uname

- Weights and Biases version: 0.10.20
- Python version: 3.6.8
- Operating System: Ubuntu 18.04 LTS

I train and test my network 10 times in a run and log `test_acc_avg` at every end of training. I set `test_acc_avg` as sweep metric and want to stop early when this run produce low `test_acc_avg`. But it seems hyperband do nothing that every run still train 10 times.

Can you help me?

```yaml
program: train.py
method: bayes
metric:
  name: test_acc_avg
  goal: maximize
early_terminate:
  type: hyperband
  eta: 2
  max_iter: 10
  s: 3
```

The hyperparameter space contains 3600 combinations and the sweep have ran 11 runs now.

**Comment by ariG23498** [comment]
Hey @hainanxzg 
Could you share the code snippet to help us reproduce the issue? That way it would be really helpful for us to give you insights into what went wrong. Thanks!

**Comment by hainanxzg** [comment]
In my train.py script:

test_acc_avg = test_f1_avg = 0
for i in range(repeats):
    test_acc, test_f1 = model.train()
    test_acc_avg += test_acc
    test_f1_avg += test_f1 
    wandb.log({"test_acc_avg ": test_acc_avg / (i+1)})

**Comment by cvphelps** [comment]
@hainanxzg the hyperband early terminator runs periodically, every few minutes, and checks what runs to terminate. This is likely why you're not seeing early termination on a very short script. @ariG23498 could you please add this note to the hyperband docs.

**Comment by hainanxzg** [comment]
@cvphelps Sorry, but the model.train() actrually run 20 mins.

**Comment by sydholl** [comment]
In the past year we've majorly reworked the CLI and UI for Weights & Biases. We're closing issues older than 6 months. Please comment to reopen.
